packages/fluentui/react-northstar/src/components/Attachment/Attachment.tsx

```diff
@@ -140,12 +140,16 @@ const Attachment = compose<'div', AttachmentProps, AttachmentStylesProps, {}, {}
       _.invoke(props, 'onClick', e, props);
     };
 
+    const mergeShorthandVariables = (variables, shorthandVariables) => {
+      return (variables || shorthandVariables) && mergeComponentVariables(variables, shorthandVariables);
+    };
```

Comment:
At least let's move this out of render scope and add a comment to do something with as obviously this about is not a key to the bright future

Reply:
Moved and added comment. I agree that we should wither fix mergeComponentVariables or provide similar utility